mail/views.py

```diff
@@ -73,6 +74,26 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
         )
 
 
+class AutomaticEmailView(GenericAPIView):
```

Comment:
Probably you can simplify the view using a different rest framework generic view (like `ListAPIView`).
If you want to leave as is, please define the serialize at the class level and retrieve in the `get` method using `self.get_serializer` like the view above.